src/splitInteger.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable strict */
```

Comment:
It's not recommended to disable eslint rules globally. If there is a specific rule that conflicts with your code, consider disabling that specific rule for the particular line or block of code 📝